Applications/Spire/Source/Ui/ListView.cpp

```diff
@@ -207,11 +207,19 @@ void ListView::append_query(const QString& query) {
   if(!m_items.empty()) {
     auto start = m_current_model->get_current().get_value_or(-1);
     auto i = (start + 1) % static_cast<int>(m_items.size());
+    auto is_repeated_query = m_query.count(m_query.at(0)) == m_query.count();
+    auto short_match = boost::optional<int>();
```

Comment:
Remove ```boost::```. It's already being used via ```using namespace boost;```. Only ```std::``` goes unused because it results in a lot of name clashes and because ```std``` is fairly short.